lib/fluent/diagtool/diagutils.rb

```diff
@@ -49,13 +49,21 @@ def initialize(params)
 	      "netstat -plan",
 	      "netstat -s",
       ]
-      if fluent_package?
+      if @conf[:type].downcase == "fluentd" && fluent_package?
         @conf[:package_name] = "fluent-package"
         @conf[:service_name] = "fluentd"
       else
         @conf[:package_name] = "td-agent"
         @conf[:service_name] = "td-agent"
       end
+
+      if @conf[:type].downcase == "fluentbit" && fluentbit_package?
+        @conf[:package_name] = "fluent-bit"
+        @conf[:service_name] = "fluent-bit"
+      else
+        @conf[:package_name] = "td-agent-bit"
+        @conf[:service_name] = "td-agent-bit"
+      end
```

Comment:
It seems that we should fix 2 points here.

* Logic branching seems wrong. the values would be always `fluent-bit` or `td-agent-bit` finally.
* We don't need to use `downcase`.

https://github.com/fluent/diagtool/blob/dce2f3eba9d5a2079167db32b056f5e04d97fe20/lib/fluent/diagtool/diagutils.rb#L272-L277

I think we should not use `@conf` for this in the first place.
We should use getters for `package_name` and `service_name` because the values are not configured by users, but they are automatically defined by the `type` option and the environment.
But it would be a large fix, so maybe we don't have to do it now.

```suggestion
      if @conf[:type] == "fluentd"
        if fluent_package?
          @conf[:package_name] = "fluent-package"
          @conf[:service_name] = "fluentd"
        else
          @conf[:package_name] = "td-agent"
          @conf[:service_name] = "td-agent"
        end
      else
        if fluentbit_package?
          @conf[:package_name] = "fluent-bit"
          @conf[:service_name] = "fluent-bit"
        else
          @conf[:package_name] = "td-agent-bit"
          @conf[:service_name] = "td-agent-bit"
        end
      end
```

Reply:
Thanks for your suggestion. It would be great to use these suggested lines.
BTW, in this case, you shouldn't use "i think" to insist your opinions. You'd batter to say: "I believe" for it.

Reply:
Thanks for your advice on my English! :smile: 
 I use many expressions without understanding the nuances, so I appreciate it very much!